store/datastore.go

```diff
@@ -42,6 +42,9 @@ type DataStore interface {
 	DoDevicesExistByIntegrationID(context.Context, uuid.UUID) (bool, error)
 	// RemoveDevicesFromIntegration integration with given id from all devices
 	// belonging to the tenant within the context.
+	RemoveIntegrationFromDevices(ctx context.Context, id uuid.UUID) (count int64, err error)
+	// UsertDeviceIntegrations adds the list of integration IDs to the
+	// device and creates it if it does not exist.
```

Comment:
RemoveDevicesFromIntegration does not match `RemoveIntegrationFromDevices` and a small misspell in UsertDeviceIntegrations

Reply:
LOL, the description is right though :smile: